.travis.yml

```diff
@@ -2,12 +2,19 @@ language: scala
 jdk: oraclejdk7
 sudo: false
 before_install: umask 0022
+cache:
+  directories:
+   - $HOME/.scalding.dep.cache
 script:
   - "echo no op"
+addons:
+  apt:
+    packages:
+    - md5deep
 matrix:
   include:
 #BASE TESTS
```

Comment:
I think that's fine. We can also run that bash script as part of the tests, so the tests use the .travis.yml and the Build.scala as inputs before we run any of the other tests it makes sure we are going to run all the tests.